build/dependencies.props

```diff
@@ -17,6 +17,6 @@
     <SystemSecurityClaimsVersion>4.3.0</SystemSecurityClaimsVersion>
     <SystemSecurityCryptographyCngVersion>4.5.0</SystemSecurityCryptographyCngVersion>
     <SystemXmlXmlDocumentVersion>4.3.0</SystemXmlXmlDocumentVersion>
-    <WCFVersion>5.0.0-ci.20229.1</WCFVersion>
+    <WCFVersion>5.0.0-dev.20229.1</WCFVersion>
```

Comment:
I'm using a dev build here because I had to manually produce a WCF build that had both Brent's recent changes and mine present. Once those PR's are merged, this can go back to being a ci build.